backend/spec/factories/patterns.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :pattern do
+    encrypted_user_id { create(:user).encrypted_id }
+    start_at { Time.zone.now }
+    end_at { Time.zone.now + 2.days.from_now }
```

Comment:
I _think_ you can just do `2.days.from_now`

Reply:
Ha! Tried it out and there's a funny behavior.

```
Time.zone.now + 2.days.from_now
=> Sat, 27 Feb 2077 05:37:16.850969000 UTC +00:00
2.days.from_now
=> Tue, 01 Aug 2023 14:48:42.233673000 UTC +00:00
```

I think `Time.zone.now + 2.days` is fine, or `2.days.from_now` which reads better to me